components/molecule/dropdownOption/src/index.js

```diff
@@ -7,31 +7,51 @@ const BASE_CLASS = 'sui-MoleculeDropdownOption'
 const CLASS_CHECKBOX = `${BASE_CLASS}-checkbox`
 const CLASS_TEXT = `${BASE_CLASS}-text`
 const CLASS_DISABLED = `${BASE_CLASS}--disabled`
+const CLASS_HIGHLIGHTED = `is-highlighted`
+const CLASS_HIGHLIGHTED_MARK = `${BASE_CLASS}--mark`
```

Comment:
I see it nice, but then this class name should be a sub-component, not a modifier: `${BASE_CLASS}-mark`.